src/data/components/group.js

```diff
@@ -36,6 +36,7 @@ const Group = React.createClass({
           onChange: (event) => {
             this.setState({newTitle: event.target.value});
           },
+          onClick: event => event.stopPropagation(),
```

Comment:
True! Feel free to add `arrow-parens` as an error in this PR or I'll add it afterwards. :)